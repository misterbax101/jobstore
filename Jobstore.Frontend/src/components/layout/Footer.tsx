import React, { Component } from 'react';
import { Container, Row, Col } from 'reactstrap';


const footerStyles: React.CSSProperties = {
    position: 'absolute',
    bottom: '0',
    width: '100%',
    minHeight: '60px',
}

class Footer extends Component {
    render() {
        const currentYear = new Date().getFullYear();
        return (
            <footer
                className="bg-light text-muted"
                style={footerStyles}>
                <Container className='py-3'>
                    <Row >
                        <Col>
                            Â© {currentYear} Jobstore  All Rights Reserved
                            <span className='float-right'>
                              Contact us 
                              <a href='mailto:' className='ml-1'>info@jobstore.com</a>
                            </span>
                        </Col>
                    </Row>
                    <hr></hr>
                </Container>
            </footer>
        )
    }
}

export default Footer;
