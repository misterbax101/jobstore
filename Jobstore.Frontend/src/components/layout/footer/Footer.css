footer.footer {
    position: fixed;
    bottom: 0;
    width : 100%;
    min-height: 60px;
}