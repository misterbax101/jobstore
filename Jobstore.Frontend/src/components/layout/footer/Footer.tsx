import React, { Component } from 'react';
import { Container, Row, Col } from 'reactstrap';
import './Footer.css';

class Footer extends Component {
    render() {
        const currentYear = new Date().getFullYear();
        return (
            <footer
                className="bg-light text-muted footer">
                <Container className='py-3'>
                    <Row >
                        <Col>
                            Â© {currentYear} Jobstore  All Rights Reserved
                            <span className='float-right'>
                              Contact us 
                              <a href='mailto:' className='ml-1'>info@jobstore.com</a>
                            </span>
                        </Col>
                    </Row>
                    <hr></hr>
                </Container>
            </footer>
        )
    }
}

export default Footer;
