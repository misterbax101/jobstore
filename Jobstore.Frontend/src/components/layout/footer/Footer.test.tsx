import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import Footer from './Footer';

describe('<Footer />', () => {
    let wrapper: ShallowWrapper<Footer>;
    beforeEach( () => {
        wrapper =  shallow(<Footer />)
    }) 

    it('should render footer', () => {
        expect(wrapper.getElements()).toMatchSnapshot()
    });
});
