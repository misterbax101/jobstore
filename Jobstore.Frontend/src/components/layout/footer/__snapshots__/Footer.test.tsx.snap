// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render footer 1`] = `
Array [
  <footer
    className="bg-light text-muted footer"
  >
    <Container
      className="py-3"
      tag="div"
    >
      <Row
        tag="div"
      >
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          Â© 
          2019
           Jobstore  All Rights Reserved
          <span
            className="float-right"
          >
            Contact us
            <a
              className="ml-1"
              href="mailto:"
            >
              info@jobstore.com
            </a>
          </span>
        </Col>
      </Row>
      <hr />
    </Container>
  </footer>,
]
`;
